package.json

```diff
@@ -21,12 +21,13 @@
     "graphology-layout": "^0.6.1",
     "graphology-layout-forceatlas2": "^0.10.1",
     "lodash": "^4.17.21",
-    "pnc-api-types-ts": "project-ncl/pnc-api-types-ts#v2.6.5-pnc-2.6.5",
+    "pnc-api-types-ts": "project-ncl/pnc-api-types-ts#v2.7.0-SNAPSHOT-pnc-8ade3a9",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-router-dom": "^6.12.1",
     "react-scripts": "5.0.1",
     "react-test-renderer": "^17.0.2",
+    "resize-observer-polyfill": "^1.5.1",
```

Comment:
Hi @patrikk0123 , why this change is required?

Reply:
tests were failing

Reply:
I am providing error details here:

![image](https://github.com/project-ncl/pnc-web-ui-react/assets/17566891/42f1c8a6-cbd6-4f4b-8f69-402cf3449324)

The only one `resize-observer-polyfill` occurence was removed, see `package-lock.json`:

![image](https://github.com/project-ncl/pnc-web-ui-react/assets/17566891/896b87a1-1eb4-428d-874f-b4e460981444)